code/src/containers/forms/new-product-form/NewProductForm.scss

```diff
@@ -0,0 +1,138 @@
+@import "@design-system";
+
+.product-form {
+  display: grid;
+  grid-template-columns: calc(30% - 10px) calc(70% - 10px);
+  grid-gap: spacing(5);
+  box-sizing: border-box !important;
+  margin-bottom: spacing(5);
+
+  & div {
+    box-sizing: border-box !important;
+  }
+
+  &__header {
+    background-color: var(--products-table-head-bg-color, $light-alpha);
+    padding: spacing(5) spacing(8);
+  }
+
+  &__body {
+    padding: spacing(8);
+  }
+
+  &__container {
+    box-shadow: var(--products-table-shadow, $shadow-black);
+    background-color: white;
+  }
+
+  &__image-preview {
+    border: 1px solid var(--spa-accordion-container-border-color, $gray-light);
+    border-radius: 3pxs;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: spacing(5);
+    aspect-ratio: 1 / 1;
+  }
+
+  &__image-section {
+    grid-column-start: 1;
+    grid-column-end: 2;
```

Comment:
Can we use shorthand here like `grid-column: 1/2`; What is your opinion on it? Also check other places

Reply:
Thanks, I used suggested approach here and in other places